#
gripper_widths:
  on_x: 52
  on_y: 32
  on_z: 10
  
#force in N
desidered_force: 8.0

#speed in mm/s
gripper_requested_speed: 50.0

#width in mm
max_gripper_width: 68.0

